mysql-test/suite/rocksdb.rpl/t/suite.opt

```diff
@@ -1 +1 @@
-$ROCKSDB_OPT $ROCKSDB_LOAD_ADD
+$ROCKSDB_OPT $ROCKSDB_LOAD_ADD --loose-rocksdb_unsafe_for_binlog=true
```

Comment:
OK, so I reworked a few of these that need the rocksdb_unsafe_for_binlog=true which was mostly splitting a few of the tests into a row and a stm_mixed test, re-recorded and a new jenkins run here http://jenkins.percona.com/view/5.7/job/mysql-5.7-param/1052/

This looks pretty good as is so I will go ahead and merge so I can continue rebasing/fixing/merging forward.